src/Sit.scala

```diff
@@ -52,6 +64,18 @@ class Sit(private val projectPath: String,
    */
   def commit(commitMessage: String): Unit = {
     // TODO
+    val currCommit = head match {
+      case Some(commit) => commit.addNext(
+        diff().getOrElse(emptyDiff),
+        commitMessage)
+      case None => Commit(
+        randomUUID().toString,
+        None,
+        diff().getOrElse(emptyDiff),
+        commitMessage,
+        0)
+    }
+    persisToDisk(currCommit)

```

Comment:
Do you think this commit is good enough?

Reply:
I think that should be good enough. But in this case we have to initialize Sit after every commit or otherwise the head will be wrong. 

That won't be an issue since I can imagine how this will be called: wrapped the compiled files with a shell and invoke it like a git command.